pkg/network/netlink/testutil/conntrack.go

```diff
@@ -138,3 +149,44 @@ func TeardownCrossNsDNAT(t *testing.T) {
 	}
 	nettestutil.RunCommands(t, cmds, true)
 }
+
+func _curDir() (string, error) {
```

Comment:
These two functions are already in the `http` package. Should we extract them to somewhere accessible to more tests?